mesonbuild/build.py

```diff
@@ -39,7 +38,10 @@
     is_known_suffix
 )
 from .linkers import StaticLinker
-from .interpreterbase import FeatureNew
+from .interpreterbase import FeatureNew, TYPE_nkwargs, TYPE_nvar
+import mesonbuild
+
+from mesonbuild import mesonlib
```

Comment:
this looks odd too

Reply:
vscode auto-import probably. Anyway, it is gone now.